### PR TITLE
Javascript/javascript1/week4

### PR DESCRIPTION
**Pull request by ehsanesmaeil**
sorry for such a delay.

### CODE REVIEW

**Review by nmoskaleva — COMMENTED** [review]
Good job on this homework! Ideally, you could write independent functions for different tasks – add a todo, perform simple math etc. Just not to write all the logic inside an if statement.

**Comment by ehsanesmaeil** [comment]
> Good job on this homework! Ideally, you could write independent functions for different tasks – add a todo, perform simple math etc. Just not to write all the logic inside an if statement.

**Comment by ehsanesmaeil** [comment]
Thanks for your considerations. :)

File: javascript/Javascript1-week4-exercise-1.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        return "Benjamin, your name is saved.";
+    } else if (command == "Hello my name is Benjamin" && name["name"] != "Benjamin") {
+        name["name"] = "Benjamin";
+        return "Nice to meet you Benjamin";
+    } else if (command == "what is my name?" && name["name"] == "Benjamin") {
+        return "Your name is Benjamin"
+    } else if (command == "what is my name?" && name["name"] == null) {
+        return "The name has not been mentioned"
+    } else if (command == "Add fishing to my todo") {
+        todo["exercise"] = "fishing"
```
[the remark this answers is not in the document]
Comment (ehsanesmaeil) [reply javascript/Javascript1-week4-exercise-1.js:31]: Ahah, Thanks, got it.

**Pull request merged**